CITATION.cff

```diff
@@ -0,0 +1,159 @@
+# schema: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+cff-version: 1.2.0
+
+title: "BIDS Stats Models Specification"
+
+version: 0.1.0
```

Comment:
not sure if the version is correct

Reply:
Probably should be 1.0.0-rc2, since 1.0.0-rc1 was the Google Doc from 2021. Unless we're ready to call 1.0.0.

Also note: https://github.com/bids-standard/stats-models/issues/48